### DISCUSSION TITLE
Rotten Tomatoes User Scores

### DISCUSSION THREAD

**Discussion by Bellgore** [sct/overseerr#3328] [Feature Requests]
Would it be possible to get the RT users score as well as the critics scores, i find the users scores to be a better indicator of how good or bad a movies is.

**Comment by KingAndaval** [comment]
This is a fair request.

What I tend to find is:

Recent movies: Critic Scores are more useful
Old movies: User scores are more useful

People tend to 'review bomb' things like Captain Marvel, but cult films with poor critic scores tend to see their audience score grow over time. 

I think part of the problem is the audience's perception of what the RT percentage means. 100% isn't 10/10... it's that all critics gave it a positive review (which could well be 6/10 depending on their editorial decisions.
